addons/l10n_hk/models/res_bank.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import re
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools import single_email_re
+
+
+auto_mobn_re = re.compile(r"""^[+]\d{1,3}-\d{1,29}$""", re.VERBOSE)
```

Comment:
- This should be written in capital case (see https://github.com/odoo/odoo/blob/16.0/addons/account/models/account_report.py#L20 for instance)
- Why do you need `re.VERBOSE` if the regex is on one line (same for the 3 double quotes instead of a one double quote) ?
- Between 1 and 29, these boundaries seem extreme (1 seems too low and 29 too high). How did you pick these numbers ?

Reply:
> Between 1 and 29, these boundaries seem extreme (1 seems too low and 29 too high). How did you pick these numbers ?
Its from another module I was working actually. May I ask if it is for HK only, can I restrict it to use 852 as region code and 8 lengths?

Reply:
Not sure, some people might live in HK with a foreign phone number and what to use the feature ? Idk